### PR TITLE
Fix HBM generation for Vivado 2019.1 and older

### PR DESCRIPTION
**Pull request by jojowi**

### CODE REVIEW

File: toolflow/vivado/platform/xupvvh/plugins/hbm.tcl [hunk 1]
```diff
@@ -217,6 +217,15 @@ namespace eval hbm {
       set hbm [ create_bd_cell -type ip -vlnv xilinx.com:ip:hbm:1.0 "hbm_0" ]
       set_property -dict $hbm_properties $hbm
 
+      # Disabling the APB debug port is only possible in Vivado 2019.2 and newer
+      if { [::tapasco::vivado_is_newer "2019.2"] == 0 } {
```
Comment (cahz) [inline toolflow/vivado/platform/xupvvh/plugins/hbm.tcl:221]: Isn't the comment and the condition contradicting? `vivado_is_newer` returns 1 if it is newer.

Comment (jojowi) [reply toolflow/vivado/platform/xupvvh/plugins/hbm.tcl:221]: In Vivado 2019.2 and newer there is a configuration parameter which allows to disable the APB debug port.
In older versions this is not possible and Vivado throws an error if you do not connect it. Thus in this case it is necessary to create a connection to constant zero

Comment (cahz) [reply toolflow/vivado/platform/xupvvh/plugins/hbm.tcl:221]: Sorry, then I misunderstood it. And the old Vivado does not complain about setting the (non-existing) parameter?

Comment (jojowi) [reply toolflow/vivado/platform/xupvvh/plugins/hbm.tcl:220]: I just checked this, there Vivado throws a warning.
I changed it so the Parameter is only used if the Vivado version is 2019.2 or newer

**Pull request merged**